src/main/java/duke/main/Duke.java

```diff
@@ -0,0 +1,114 @@
+package duke.main;
+
+
+import java.util.ArrayList;
+import java.io.IOException;
+import java.util.Scanner;
+
+import duke.command.*;
```

Comment:
Consider removing this import, as the specific imports may already be covered by the lines below.